antora-ui-camel/src/css/site.css

```
@@ -29,3 +29,4 @@
 @import 'catalog.css';
 @import 'sharingbuttons.css';
 @import 'security.css';
+@import 'tabs.css'
```

Try this instead, so we don't need to copy `tabs.css` from @asciidoctor/tabs
```suggestion
@import '@asciidoctor/tabs/dist/css/tabs.css'
```